submissions/antonkottans/memory-pair-game/script.js

```diff
@@ -0,0 +1,250 @@
+const pipe = (...arrayOfFunctions) => {
```

Comment:
Is there a reason for `pipe` usage besides just to use pipe? It is used only once and makes code much more complicated.

Reply:
Becouse     
return pipe(
        getCardNames(amountOfElements),
        doubleAndShuffle,
        convertNamesToNodes
);
is more readable then
return convertNamesToNodes(doubleAndShuffle(getCardNames(amountOfElements)))